vllm/entrypoints/openai/serving_chat.py

```diff
@@ -367,7 +367,7 @@ async def chat_completion_full_generator(
                 logprobs = self._create_logprobs(
                     token_ids=token_ids,
                     top_logprobs=top_logprobs,
-                    num_output_top_logprobs=request.logprobs,
+                    num_output_top_logprobs=request.top_logprobs,
```

Comment:
There is another place where `_create_logprobs` is called with incorrect `num_output_top_logprobs`.